### ISSUE TITLE
Unable to Sign-in to the Smart FHIR Sandbox

### ISSUE THREAD

**Issue by ILindsley** [smart-on-fhir/smart-on-fhir.github.io#98]
I know I'm a total noob for asking this, and I know the answer to this is probably obvious....but whenever I click on the link "sign into our sandbox" on the http://docs.smarthealthit.org/tutorials/testing/ page, a patient search web app of some sort launches, and an error is immediately displayed.

This is the error: "Search failed, see console"

I swear I wasn't getting this error before (last week).  Essentially, I'm trying to test my app using the sandbox.  Any help would be greatly appreciated.

Ian

**Comment by jmandel** [comment]
Thanks for this report -- should be fixed now!

**Comment by ILindsley** [comment]
Josh,

I'm still getting the same error.  Here are the steps to reproduce:


   1. Navigate to http://docs.smarthealthit.org/
   2. From the "Sandbox" drop down at the top of the page select "Test Your
   App"
   3. Click the link titled "our public sandbox server
   <https://fhir.smarthealthit.org/>" which navigates me to
   https://fhir-dstu2.smarthealthit.org/#/ui/select-patient (Clicking on
   the sign into our sandbox <https://fhir.smarthealthit.org/> link has the
   same effect).  Once I have been navigated to
   https://fhir-dstu2.smarthealthit.org/#/ui/select-patient, I get the
   error show in the attached screenshot (initital_error.png).  I had the
   chrome debugger on when I captured the screenshot; this may provide you
   with some additional information.  One error I noticed is:

*GET
https://fhir-api-dstu2.smarthealthit.org/Patient?_sort:asc=given&_sort:asc=family&_count=25&name=smith
<https://fhir-api-dstu2.smarthealthit.org/Patient?_sort:asc=given&_sort:asc=family&_count=25&name=smith>
401 (Unauthorized)*

*Is this because I'm not logged in or something?*


Let me know if you need any additional information.  Thanks for your help
yesterday, and your speedy response.

Ian

**Comment by nschwertner** [comment]
Hmm. Could it be that your FHIR Starter still thinks that the main API server is an open one? Perhaps clearing the local storage for `fhir-dstu2.smarthealthit.org` would help? Or try a different browser or same browser in anonymous mode?

**Comment by ILindsley** [comment]
No dice.  I'm still getting the same error.

Is it correct that I'm being sent to
https://fhir-dstu2.smarthealthit.org/#/ui/select-patient after clicking on
the "sign into our sandbox" link?  I could have sworn before (last week)
that I somehow got to a different SMART FHIR starter page--where I was able
to launch several apps, including the growth chart app from the FHIR
starter.  Is there another link to get back to this page?  Or am I being
sent to the select patient widget in order to establish patient context?

Thanks for all your help.  I just can't figure out why this is happening
all of a sudden.  I cleared the browser cache and the problem is still
occurring.

Ian
